ts/features/itwallet/issuance/screens/ItwIssuanceCredentialTrustIssuerScreen.tsx

```diff
@@ -176,12 +177,18 @@ const ContentView = ({ credentialType, eid }: ContentViewProps) => {
     source: getCredentialNameFromType(eid.credentialType, "", isItwL3)
   }));
 
+  // Added hasScrolledToBottom ref to avoid sending multiple scroll-to-bottom events when navigating between screens
   const trackScrollToBottom = (crossed: boolean) => {
-    if (crossed) {
+    if (crossed && !hasScrolledToBottom.current) {
+      // eslint-disable-next-line functional/immutable-data
+      hasScrolledToBottom.current = true;
       trackIssuanceCredentialScrollToBottom(
         mixPanelCredential,
         ITW_ROUTES.ISSUANCE.CREDENTIAL_TRUST_ISSUER
       );
+    } else if (!crossed && hasScrolledToBottom.current) {
```

Comment:
We are resetting `hasScrolledToBottom` when the user scrolls up again. Is it the expected behaviour? Scrolling up and down multiple times would send the same event multiple times.